Description: use a path that's always there
 Or at least in Debian ;)
 .
 To try to use /etc/cups/ it will require to install cups first. But to be
 able to achieve the same result one can simply use /etc/crond.d, which
 is present both in Debian and Red Hat almost always (if not always).
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: https://bugs.kde.org/show_bug.cgi?id=330107
Last-Update: 2014-01-17

Index: kde-runtime/kioslave/trash/tests/testtrash.cpp
===================================================================
--- kde-runtime.orig/kioslave/trash/tests/testtrash.cpp	2015-02-23 22:55:56.275954508 +0100
+++ kde-runtime/kioslave/trash/tests/testtrash.cpp	2015-02-23 22:55:56.231955988 +0100
@@ -894,8 +894,8 @@
 void TestTrash::trashDirectoryOwnedByRoot()
 {
     KUrl u;
-    if ( QFile::exists( "/etc/cups" ) )
-        u.setPath( "/etc/cups" );
+    if ( QFile::exists( "/etc/cron.d" ) )
+        u.setPath( "/etc/cron.d" );
     else if ( QFile::exists( "/boot" ) )
         u.setPath( "/boot" );
     else
